src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +294,38 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(IEnumerable<NuGetVersion> bundleVersions, NuGetVersion currentMatchingVersion)
+        {
+            bool isStandardOrExtendedChannel = string.Equals(_platformReleaseChannel, ScriptConstants.StandardPlatformChannelName, StringComparison.Ordinal) ||
+                                   string.Equals(_platformReleaseChannel, ScriptConstants.ExtendedPlatformChannelName, StringComparison.Ordinal);
+
+            bool isLatestOrEmptyChannel = string.Equals(_platformReleaseChannel, string.Empty, StringComparison.Ordinal) ||
+                                          string.Equals(_platformReleaseChannel, ScriptConstants.LatestPlatformChannelName, StringComparison.Ordinal);
+
+            if (isStandardOrExtendedChannel)
```

Comment:
instead of the bool checks, can we break the logic down into re-usable components, and then call the appropriate one given the channel? Can even use a switch statement for that:


```
return _platformReleaseChannel.ToUpperInvariant() switch
{
    "STANDARD" or "EXTENDED" => // call something
    "LATEST" or "" => // call something
    _ => // call something
}
```

Reply:
You might be able to also conver the entire method to a switch

```
        private NuGetVersion ResolvePlatformReleaseChannelVersionTest(IEnumerable<NuGetVersion> bundleVersions, NuGetVersion currentMatchingVersion) => _platformReleaseChannel switch
        {
            "STANDARD" or "EXTENDED" => // call something,
            "LATEST" or "" => // call something,
            _ => // call something
        };
```

Reply:
I combined the creation of the ordered bundles version list with the switch statement into one method based on another piece of feedback earlier (sorry, didn't see the second comment in time). I think it's alright to keep since it groups the reordering of the bundle versions and the decision-making based on the ordered list close together. 

Currently still using `ScriptConstants` file to store the known channel names, which is less readable, but given the broad name of this setting I think it could be used in other places eventually.